_posts/2021-3-26-FFX-20-years-later.md

```diff
@@ -0,0 +1,51 @@
+
+I also liked how your characters are really rewarded for leveling up. Whenever you get to a zone, your counter for a random enemy might be just out of range to be 1 shot. But by the end of the zone you'd almost always hit that bar. It always felt like significant progression.
+
+I really enjoyed the Conditional Turn Based Battle System in FFX. Rather than having a turn where everyone gets to go and your characters speed determines who goes first in the order (think: pokemon). In FFX, your character's speed effects how long it takes for them to go again. So you could have Tidus make 2 or 3 moves between every enemy's move. Also certain moves take longer to recharge. For example Firaga (Fire 3) takes longer to recharge than Fire (Fire 1), so there's an incentive to use a smaller spell if you only need a smaller spell (normally mana is the incentive, but it never ends up being important late game).
+
+I think this system works really well every so often, but it's biggest downside is that to be successful, you really have to exploit it as much as possible with Haste and Slow spells. There are bosses late game that are immune to slow and remove haste and honestly those felt the most balanced and challenging.
+
+The sphere grid system removed random stat gains and always got you excited to move 3 spheres forward. I really loved this system and wish it was more prevalent in games, albeit a little simplified/streamlined. It also got you excited about hitting a strength boost for Wakka/Tidus/Auron becuase 4 strength is HUGE in this game. 
+
+As far as bosses were concerned, JRPGs always have 3 different kinds, puzzle bosses, standard bosses, and dps checks. There was only one instance of the latter and it really only occurred at the end of the game, which gave you the confidence that your characters were strong enough to beat the game (I'm not sure if this was actually true, but it felt like it). The puzzle bosses were interesting, most of them felt like you could figure them out after a few turns of trial and error, watching patterns. If you did need more time and ended up dying, the cutscenes around them were pretty quick.
+
+I think going into the last boss, I was severely underleveled because the first time I died, it didn't feel close. I contemplated just giving up and watching the end on youtube. After looking at guides and seeing the builds that they had suggested, it was very apparent that you were expected to be at the end of the sphere grid for almost every party member and I had only hit the end for Auron. I decided to just try and refine my strategy and after 4 hours of trying and failing (most of the time was because unskippable cutscenes took no joke 25 minutes), I finally came to a strategy that worked and I felt like I had to really master and understand the systems to do so. The last hit using Bahamut's Limit break was extremely satisfying and filled me with adrenaline.
+
+## Story and Score
+
```

Comment:
Expand on conflict of future and past with regards to religion through wakkas lens.